secrets/sso-app-secret.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Is this required?  I suspect it should be provided by the sso deliverables.

Reply:
Agreed, I'll remove and fetch from SSO.